ecs-cli/modules/clients/aws/kms/mock/client.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2015-2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// Copyright 2015-2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
the fact this year is going backwards makes me suspicious that something is not up to date re: mocks

Reply:
hmmm but wait, other mocks that are updated seem to retain the `2017` year... @_@